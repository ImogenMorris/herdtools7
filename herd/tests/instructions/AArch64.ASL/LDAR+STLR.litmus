AArch64 LDAR+STLR
{
  int64_t x=2;
  int64_t y=0;
  0:X0=x;
  0:X2=y;
}
  P0          ;
 LDAR X1,[X0] ;
 STLR X1,[X2] ;

forall [y]=2
