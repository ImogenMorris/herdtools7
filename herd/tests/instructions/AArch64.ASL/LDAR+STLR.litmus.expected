Test LDAR+STLR Required
States 1
[y]=2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([y]=2)
Observation LDAR+STLR Always 1 0
Hash=95e2ee017cb0a4683eef6895ed9b6a3f

